tests/asr_test.py

```
@@ -21,119 +22,44 @@ def test_config_creation(self, asr_config):
-        # Test with different chunk size
-        config = ASRConfig(model_id='openai/whisper-large-v3-turbo', seconds_per_chunk=5, batch_size=8)
-        model = ASRonSPEED(config=config)
-
-        assert model.config.seconds_per_chunk == 5
-        assert model.config.batch_size == 8
-        assert not model.is_warmed_up
-
-    def test_device_selection(self):
-        """Test device selection logic"""
-        import torch
+    def test_quality_of_transcription(self, asr_model, test_audio_files):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_quality_of_transcription`

Add return type annotation: `None`

(ANN201)